src/OceanSimulations/prescribed_ocean_model.jl

```diff
@@ -0,0 +1,96 @@
+using Oceananigans.Models: AbstractModel
+using Oceananigans.OutputReaders: extract_field_time_series, update_field_time_series!
+
+import Oceananigans.TimeSteppers: time_step!, update_state!, reset!, tick!
+import Oceananigans.Models: timestepper, update_model_field_time_series!
+
+import ClimaOcean: reference_density, heat_capacity
+import Oceananigans.Architectures: on_architecture
+
+#####
+##### A prescribed ocean...
+#####
+
+struct PrescribedOceanModel{G, C, U, T, F, Arch} <: AbstractModel{Nothing, Arch}
```

Comment:
We call it `PrescribedAtmosphere`. Is this `PrescribedOcean` or `PrescribedOceanModel`?

I'm not sure really sure what a "prescribed model" is.

Reply:
Hmm, this is because we add a simulation on top, while the atmosphere is a "simulation" rather than a model. I can switch it to a simulation rather than a model